source/content/guides/new-relic/04-new-relic-quicksilver.md

```diff
@@ -17,98 +17,15 @@ permalink: docs/guides/new-relic/new-relic-quicksilver
-
-1. Navigate to the project's root directory. You can use Terminus to provide the clone command:
-
-  ```bash{outputLines:2-3}
-  terminus connection:info $site.dev --fields='Git Command' --format=string
-
-  git clone ssh://codeserver.dev.2187...d85b@codeserver.dev.2187...d85b.drush.in:2222/~/repository.git sitename
-  ```
-
-1. Copy and run the output command and then change directory to the site code root:
-
-  ```bash{promptUser: user}
-  cd $site
-  ```
-
-1. Set the connection mode to Git:
-
-  ```bash{promptUser: user}
-  terminus connection:set $site.dev git
-  ```
-
-1. Create a copy of [Pantheon's `new_relic_deploy.php`](https://github.com/pantheon-systems/quicksilver-examples/blob/master/new_relic_deploy/) script in the project's private path:
-
-  ``` bash{promptUser: user}
-  mkdir -p private/scripts
-  curl https://raw.githubusercontent.com/pantheon-systems/quicksilver-examples/master/new_relic_deploy/new_relic_deploy.php --output ./private/scripts/new_relic_deploy.php
-  ```
-
-1. Create a `pantheon.yml` file if one doesn't already exist in your root directory.
-
-1. Paste the following workflow into your `pantheon.yml` file to hook into the platform after code is synced on Dev/Multidev and deployed to Test/Live to fire off the New Relic&reg; integration script:
-
-  ```yaml:title=pantheon.yml
-  api_version: 1
-
-  workflows:
-    # Log to New Relic when deploying to test or live.
-    deploy:
-      after:
-        - type: webphp
-          description: Log to New Relic
-          script: private/scripts/new_relic_deploy.php
-    # Also log sync_code so you can track new code going into dev/multidev.
-    sync_code:
-      after:
-        - type: webphp
-          description: Log to New Relic
-          script: private/scripts/new_relic_deploy.php
-  ```
-
-   <Alert title="Note" type="info">
-
-   `api_version` should be set once in [`pantheon.yml`](/pantheon-yml). If you have an existing `pantheon.yml` with this line, do not add it again.
-
-   </Alert>
-
-1. [Add, commit, and push](/guides/git/git-config#push-changes-to-pantheon) changes to the Dev/Multidev environment:
-
-  ```bash{promptUser: user}
-  git add private/scripts/new_relic_deploy.php
-  git commit -am "Adding deployment recording to New Relic"
-  git push origin master
-  ```
-
-  In the terminal, you should see that your `pantheon.yml` file is being applied. Even this initial code push should appear in your Dev environment's New Relic&reg; account, on the **Deployments** tab:
-
-  ![Deployment tab display](../../../images/integrations/newrelic/deploy_tab.png)
-
-1. Confirm that there are no issues, then deploy your new commit to Test and Live. Your deploys will now be recorded in New Relic&reg; Performance Monitoring.
+To add these labels automatically on code pushes or deployments, follow the steps outlined in the [script documentation](https://github.com/pantheon-systems/quicksilver-examples/blob/main/new_relic_deploy/README.md).
```

Comment:
```suggestion
To add these labels automatically on code pushes or deployments, follow the steps outlined in the [script documentation](https://github.com/pantheon-systems/quicksilver-examples/blob/main/new_relic_deploy).
```

Chatting with @rachelwhitton right now, we think it's better to link to the directory.